app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+end
```

Comment:
- [ ] Good job on creating the `user_controller` 👏 . Kindly add the `show` action as specified in the project requirements. note, It's okay for the actions to be empty for now. 

### Sample code
```
class UsersController < ApplicationController
  
  def show;
  end

end
```

### Routes needed:
![image](https://github.com/Otegaa/blog-app/assets/50990925/8564b473-5ea8-4e43-bb4a-d05eddd20c68)